### ISSUE TITLE
Keylights occasionally become unresponsive

### ISSUE THREAD

**Issue by mariusstrom** [derjayjay/homebridge-keylights#6]
Not sure if this is just a Keylight behavior issue, but the Keylights occasionally become unresponsive in Homebridge. I have to powercycle the keylight to get it responsive again. This behavior is new since I've set up the homebridge-keylights plugin, it was working fine via Elgato Control Center before.

Not sure if the plugin is sending commands, not closing connections, etc. that could cause the device to hang?

HOOBS 3.2.9, Node 12.16.3, Keylights 1.2.0.

**Comment by derjayjay** [comment]
Yes, this seems to be an issue with the Keylights and power cycling is the only workaround we know.

Somewhat amusing, I had the complete opposite experience: not responding with the Control Center (and even not showing up at all), but no issues with Homebridge.

What you can try: there was a firmware update some weeks ago, maybe that helps? Also, does the light still work in the Control Center app when it becomes unresponsive in Homebridge?

**Comment by mariusstrom** [comment]
@derjayjay my Keylight is fully firmware updated so I think that's a non-issue. I'll validate the Control Center failure today - but I'm pretty sure I tried that last week and the Keylight was fully unresponsive. I suspect it's a Keylight issue itself, but also suspecting that the homebridge-keylights plugin exascerbates the issue as I wasn't having this problem before I set it up.

Will update the issue once I get a repro, should have something in ~24 hours if history holds up. Worst case I'll get a little wemo switch on it and schedule a 3am "powercycle the light" automation.

**Comment by mariusstrom** [comment]
@derjayjay took 36 hours but keylight is zonked. It's not discoverable from Control Center in this scenario as well.

**Comment by derjayjay** [comment]
Ah, that's a bummer. You could try reduce the polling rate in plug-in configuration, maybe that helps.
